devel/meson/Portfile

```diff
@@ -68,7 +68,8 @@ if {${os.platform} eq "darwin" && ${os.major} <= 13} {
 }
 
 # https://trac.macports.org/ticket/63462
-patchfiles-append    patch-meson-objc-accept-gnu89.diff
+# no longer needed as v0.59.3 seems to includes objc gnu89 support
+#patchfiles-append    patch-meson-objc-accept-gnu89.diff
```

Comment:
If no longer needed, remove these lines and the patchfile.

Reply:
Hi @ryandesign,

I have made these changes and updated my commit.

Thanks,

-ranga